src/main/java/appeng/core/AEConfig.java

```diff
@@ -93,6 +93,7 @@
 	private final int[] craftByStacks = { 1, 10, 100, 1000 };
 	private final int[] priorityByStacks = { 1, 10, 100, 1000 };
 	private final int[] levelByStacks = { 1, 10, 100, 1000 };
+	private final int[] levelByMillibuckets = { 10, 100, 1000, 10000 };
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1170)